inst/fw_templ/p_example/ui_body.R

```diff
@@ -141,6 +141,47 @@ application_setup <- tabItem(tabName = "application_setup",
 plot2_hover <- hoverOpts(id = "examplePlot2_hover")
 
 
+react_table_box <- box(
+    id          = "react_downloader",
+    title       = "React Table Downloader",
+    status      = "info",
+    solidHeader = TRUE,
+    collapsible = TRUE,
+    width       = 12,
+    fluidRow(column(width = 6,
+                    tags$dl(tags$dt("Features"),
+                            tags$ul(tags$li("React table downloader module display tabular data in rich formatted tables using", tags$b("`reactable`"), "package"),
+                                    tags$li("downloadabkeReactTable returns two names reactive list that can be used in application different areas",
```

Comment:
downloadableReactTable returns a named reactive list that can be used in the application.